frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -172,10 +211,13 @@ export default {
     },
     getWorkers () {
       const workers = map(this.internalWorkers, internalWorker => {
-        return omit(internalWorker, ['id', 'valid'])
+        return omit(internalWorker, ['id', 'valid', 'isNewWorker'])
       })
       return workers
     },
+    getZonesNetworkConfiguration () {
```

Comment:
Why do we need this method? We have the computedProperty.

Reply:
I guess we don't need it probably a leftover. I removed it and will test properly